server/schedule/operator/builder.go

```diff
@@ -373,39 +550,44 @@ func (b *Builder) allowLeader(peer *metapb.Peer) bool {
 // 2. add learner + remove learner.
 // 3. add learner + promote learner + remove voter.
 // 4. promote learner.
-// 5. remove voter/learner.
-// 6. add voter/learner.
+// 5. demote voter.
```

Comment:
should consider promote learner + demote voter as a step?

Reply:
Makes sense.

Reply:
Or you can change later since it is not used anyway.

Reply:
good catch.

Reply:
and also `add voter + demote voter + remove learner`

Reply:
Will be updated in the future PR.